### PR TITLE
Updated ami's and packages and added x2go provisioner

### PR DESCRIPTION
**Pull request by th1vairam**

### CODE REVIEW

**Comment by kdaily** [comment]
@th1vairam says close, so I close.

File: config.json [hunk 1]
```diff
@@ -9,11 +9,11 @@
             "type": "amazon-ebs",
             "access_key": "{{user `aws_access_key`}}",
             "secret_key": "{{user `aws_secret_key`}}",
-            "region": "us-east-1",
-            "source_ami": "ami-9711c6fc",
+            "region": "us-west-2",
```
Comment (kdaily) [inline config.json]: No us-west. Defeats speed increases (and fees) for Synapse data transfer (in us-east)

File: config.json [hunk 2]
```diff
@@ -9,11 +9,11 @@
...
+            "source_ami": "ami-f35e56c3",
             "ssh_username": "ec2-user",
             "ami_name": "CentOS 6.6 - SageCommonCompute {{timestamp}}",
-            "instance_type": "c3.4xlarge"
+            "instance_type": "t2.large"
```
Comment (kdaily) [inline config.json]: No t2.large. Can keep this in your own branch, please!

File: config.json [hunk 3]
```diff
@@ -37,14 +37,17 @@
         {
             "type": "file",
             "source": "./python_requirements.txt",
-            "destination": "/home/ec2-user"
+            "destination": "/home/ec2-user/"
```
Comment (kdaily) [inline config.json]: Just a note these are actually broken now with CfnCluster release 1.0.0 (default user to login is now `centos`)

File: config.json [hunk 4]
```diff
@@ -37,14 +37,17 @@
...
         },
         {
-            "type": "shell",
-            "scripts": ["user_provisioner.sh"],
-            "pause_before": "10s",
-            "execute_command": "{{ .Vars }} sudo -E sh '{{ .Path }}'",
-            "only": ["amazon-ebs"]
```
Comment (kdaily) [inline config.json]: don't remove

File: config.json [hunk 5]
```diff
@@ -37,14 +37,17 @@
...
+            "type": "file",
+            "source": "./python3_requirements.txt",
+            "destination": "/home/ec2-user/"
+        },
+        {
+            "type": "file",
+            "source": "./rserver.conf",
+            "destination": "/home/ec2-user/"
```
Comment (kdaily) [inline config.json]: not for commoncompute - won't provide RStudio server there

File: config.json [hunk 6]
```diff
@@ -55,28 +58,28 @@
...
+            "scripts": ["x2go_provisioner.sh"],
             "pause_before": "10s",
             "execute_command": "{{ .Vars }} sudo -E sh '{{ .Path }}'",
             "only": ["amazon-ebs"]
         },
-        {
+	{
             "type": "shell",
-            "scripts": ["cloudbio_provisioner.sh"],
+            "scripts": ["rstudio_provisioner.sh"],
```
Comment (kdaily) [inline config.json]: no rstudio (b/c no X-windows)

File: config.json [hunk 7]
```diff
@@ -55,28 +58,28 @@
...
+            "scripts": ["r_provisioner.sh"],
             "pause_before": "10s",
             "execute_command": "{{ .Vars }} sudo -E sh '{{ .Path }}'",
             "only": ["amazon-ebs"]
         },
-        {
+	{
             "type": "shell",
-            "scripts": ["r_provisioner.sh"],
+            "scripts": ["x2go_provisioner.sh"],
```
Comment (kdaily) [inline config.json]: no x2go (no x-windows)

File: config.json (hunk 1 above)
Comment (th1vairam) [inline config.json]: Not really. Depends on the use case. If you want to store in default synapse storage may be yes. But still the data transfer speed and cost overhead across regions is very much negligible.

File: config.json (hunk 2 above)
Comment (th1vairam) [inline config.json]: Noted

**Pull request closed without merging**